### PR TITLE
Update repo list

### PR DESCRIPTION
**Pull request by CourtneyBrousseau**
Update the README to include more information about miscellaneous and deprecated code.gov repositories. There are still a few more repos that are not documented on the README (listed below) because I was not able to determine their current state/use  

I also updated the contact section to reflect the style from our new repo template and also consolidated sections to simplify the content organization

Unlisted repos:
https://github.com/GSA/code-gov-admin-backend
https://github.com/GSA/code-gov-admintool
https://github.com/GSA/code-gov-repos-parser
https://github.com/GSA/code-gov-coding-languages
https://github.com/GSA/code-gov-data-quality-poc
https://github.com/GSA/code-gov-gitsecretpatterns

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,14 +40,32 @@ please use the [GSA/code-gov](https://github.com/GSA/code-gov) repo as a catch a
...
+| [GSA/code-gov-harvester-deprecated](https://github.com/GSA/code-gov-harvester-deprecated)                               | The old version of the harvester that harvests and processes code.json files from agencies. | The new harvester repo is [GSA/code-gov-harvester](https://github.com/GSA/code-gov-harvester) |
+| [GSA/code-gov-about-page](https://github.com/GSA/code-gov-about-page)                               | Component for the About page on [Code.gov](https://code.gov). | The about page component was integrated directly into [GSA/code-gov-front-end](https://github.com/GSA/code-gov-front-end) in order to simplify the site architecture |
+| [GSA/code-gov-fscp-react-component](https://github.com/GSA/code-gov-fscp-react-component)                               | Federal Source Code Policy (FSCP) plugin for code-gov-web, built with Markdown and React. | The FSCP is hosted at [https://sourcecode.cio.gov/](https://sourcecode.cio.gov/) and it was therefore duplicative for it to be on [Code.gov](https://code.gov) as well |
```
Comment (DanielJDufour) [inline README.md]: Minor correction. code-gov-fscp-react-component was originally built for the React version of code.gov.  If my memory serves me correctly, when I originally created code-gov-fscp-react-component, the React version hadn't been named `code-gov-front-end` yet.  Once a naming decision was made, I forgot to update the description in the plugin repo Readme.  So this should read `... (FSCP) plugin for code-gov-front-end, built with Markdown and React`.  I'm sorry for the confusion.

File: README.md [hunk 2]
```diff
@@ -69,4 +77,11 @@ These repos are marked as miscellaneous because they do not affect the developme
 - [repo-labels.json](repo-labels.json): A json file that can be used to add the specific GitHub issue labels code.gov uses on their projects
 - [repository_management.md](repository_management.md): This is a living document explaining how we collaboratively manage the Code.gov platform repositories
 
-## Thanks!
+
+## Questions?
+If you have questions, please feel free to contact us:  
+* Open an issue
```
Comment (saracope) [inline README.md]: Please go ahead and link this to the new issue page for this repo. It looks like that was missed before.

File: README.md (hunk 1 above)
Comment (CourtneyBrousseau) [reply README.md]: Ah ok makes sense - I was wondering about that discrepancy so thanks for clarifying

File: README.md (hunk 2 above)
Comment (CourtneyBrousseau) [reply README.md]: I copied this text from the repo template to update it and I forgot to add the link. Just added it!

**Comment by CourtneyBrousseau** [comment]
@saracope @DanielJDufour any thoughts on whether we should include any of these 6 repos in any of the lists before we merge this? 

https://github.com/GSA/code-gov-admin-backend
https://github.com/GSA/code-gov-admintool
https://github.com/GSA/code-gov-repos-parser
https://github.com/GSA/code-gov-coding-languages
https://github.com/GSA/code-gov-data-quality-poc
https://github.com/GSA/code-gov-gitsecretpatterns

**Comment by DanielJDufour** [comment]
AFAIK, 
the following can be archived and put in deprecated list:
 - https://github.com/GSA/code-gov-admin-backend
 - https://github.com/GSA/code-gov-admintool
 - https://github.com/GSA/code-gov-repos-parser
 - https://github.com/GSA/code-gov-coding-languages
 
I'd check with @froi about:
 - https://github.com/GSA/code-gov-data-quality-poc
 - https://github.com/GSA/code-gov-gitsecretpatterns

Thanks for asking :-)

**Pull request merged**